torch/testing/_internal/common_optimizers.py

```diff
@@ -833,6 +834,100 @@ def optim_error_inputs_func_lbfgs(device, dtype):
     return error_inputs
 
 
+def optim_inputs_func_muon(device, dtype=None):
+    return [
+        OptimizerInput(params=None, kwargs={}, desc="default"),
+        OptimizerInput(
+            params=None,
+            kwargs={
+                "adjust_lr_fn": lambda lr, param_shape: lr,
+            },
+            desc="passing alternative adjust_lr_fn",
+        ),
```

Comment:
let's have configs for every arg in the constructor (weight decay, momentum, etc)